packages/compass-aggregations/src/components/pipeline-explain/explain-query-performance.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { Body, Subtitle, css, spacing } from '@mongodb-js/compass-components';
+import type { IndexInformation } from '@mongodb-js/explain-plan-helper';
+
+import { ExplainIndexes } from './explain-indexes';
+
+type ExplainQueryPerformanceProps = {
+  executionTimeMillis: number;
+  nReturned: number;
+  usedIndexes: IndexInformation[];
+};
+
+const containerStyles = css({
+  display: 'flex',
+  gap: spacing[3],
+  flexDirection: 'column',
+});
+
+const statsStyles = css({
+  gap: spacing[1],
+  display: 'flex',
+  flexDirection: 'column',
+});
+
+const statItemStyles = css({
+  display: 'flex',
+  gap: spacing[1],
+});
+
+const statTitleStyles = css({
+  whiteSpace: 'nowrap',
+});
+
+export const ExplainQueryPerformance: React.FunctionComponent<ExplainQueryPerformanceProps> =
+  ({ nReturned, executionTimeMillis, usedIndexes }) => {
+    return (
+      <div
+        className={containerStyles}
+        data-testid="pipeline-explain-results-summary"
+      >
+        <Subtitle>Query Performance Summary</Subtitle>
+        <div className={statsStyles}>
+          <div className={statItemStyles}>
+            <Body>Documents returned:</Body>
+            <Body weight="medium">{nReturned}</Body>
```

Comment:
Seems like nReturned is not always returned here, maybe we should show a placeholder or something? This is how an explain looks for me for a random aggregation I tried:

<img width="631" alt="image" src="https://user-images.githubusercontent.com/5036933/168997658-1751070e-e26c-4e96-9ac4-efd60f68f158.png">

In other places we use em dash (`—`) for that

Reply:
In explain for queries, we are using colon. Not sure if we should change it here.
But if we are in the process of transitioning over to dash throughout compass, I'd be happy to do that.

Reply:
I'm okay with any placeholder, this was just a suggestion that I saw that we use often in other places, whatever you prefer that is not just empty space would be great